OWNERS

```diff
@@ -12,3 +12,6 @@ reviewers:
  - kashifest
  - jan-est
  - xenwar
+ - Jaakko-Os
```

Comment:
Not sure if the capitalization matters. but all other are lower case.

Reply:
It doesn't I guess, the capitalized name is already in other [repository](https://github.com/metal3-io/cluster-api-provider-metal3/blob/f9f2b3394d02942821819ffcd9a3e7324a5ab037/OWNERS#L13) owners file.